evm/generators.go

```diff
@@ -1175,13 +1465,40 @@ func {{.DeployHandler.HandlerName}}() *cobra.Command {
 	cmd.Flags().Uint64Var(&gasLimit, "gas-limit", 0, "Gas limit for the transaction")
 	cmd.Flags().BoolVar(&simulate, "simulate", false, "Simulate the transaction without sending it")
 	cmd.Flags().UintVar(&timeout, "timeout", 60, "Timeout (in seconds) for interactions with the JSONRPC API")
-
+	cmd.Flags().StringVar(&safeAddress, "safe", "", "Address of the Safe contract")
+	cmd.Flags().StringVar(&safeApi, "safe-api", "", "Safe API for the Safe Transaction Service (optional)")
+	cmd.Flags().StringVar(&safeCreateCall, "safe-create-call", "", "Address of the CreateCall contract (optional)")
+	cmd.Flags().Uint8Var(&safeOperation, "safe-operation", 1, "Safe operation type: 0 (Call) or 1 (DelegateCall) - default is 1")
+	
 	{{range .DeployHandler.MethodArgs}}
 	cmd.Flags().{{.Flag}}
 	{{- end}}
 
 	return cmd
 }
+
+func generate{{.StructName}}DeployBytecode(
```

Comment:
This would also be useful for the `--ecode` flag: [issue](https://github.com/moonstream-to/seer/issues/19)

No need to address in this PR, but this strategy applies to that, as well.

Reply:
Nice catch! I will comment on the issue as a possible solution

Reply:
Updated the issue with this comment https://github.com/moonstream-to/seer/issues/19#issuecomment-2383971861